sympy/calculus/util.py

```diff
@@ -569,6 +570,76 @@ def lcim(numbers):
 
     return result
 
+def is_convex(f, *syms, **kwargs):
+    """Determines the  convexity of the function passed in the argument.
+
+    Parameters
+    ==========
+
+    f : Expr
+        The concerned function.
+    syms : Tuple of symbols
+        The variables with respect to which the convexity is to be determined.
+    domain : Interval
```

Comment:
Add `, optional` after `Interval` and maybe say something about the default value below.